src/components/data-processing/ConfigureEmbedding/CalculationConfig.jsx

```diff
@@ -28,7 +27,7 @@ const MIN_DIST_TEXT = 'This controls how tightly the embedding is allowed to com
   + 'to local structure. Expected range: 0.001 to 0.5. Default is 0.1.';
 
 const CalculationConfig = (props) => {
-  const { experimentId } = props;
+  const { experimentId, pipelineRunHandler } = props;
```

Comment:
I am failing to find where is this `pipelineRunHandler` function defined. Can you point me to the file with the definition?

Reply:
It is in src/pages/experiments/[experimentId]/data-processing/index.jsx, the page for data processing

Reply:
it has now been renamed to `onPipelineRun` btw